internal/manifest/os.go

```diff
@@ -562,7 +593,8 @@ func (p *OS) serialize() osbuild.Pipeline {
 
 	if p.SElinux != "" {
 		pipeline.AddStage(osbuild.NewSELinuxStage(&osbuild.SELinuxStageOptions{
-			FileContexts: fmt.Sprintf("etc/selinux/%s/contexts/files/file_contexts", p.SElinux),
+			FileContexts:     fmt.Sprintf("etc/selinux/%s/contexts/files/file_contexts", p.SElinux),
+			ForceAutorelabel: common.ToPtr(true),
```

Comment:
This is part of https://github.com/osbuild/osbuild-composer/pull/3239/commits/6930b567d66e0e164ed556951094d0ac96abc1f1 but I don't see any obvious reason why this change is part of that commit. Why it's needed? 🤔

Reply:
I think that the change should be completely dropped from that commit, because it is later overwritten by https://github.com/osbuild/osbuild-composer/pull/3239/commits/cf559006da6c50d36c9de0c44e581e89d2d6e7b3

Reply:
Bad rebase / squash after trying out things that ended up in other commits.  I'll amend.